packages/zoe/src/zoeService/startInstance.js

```diff
@@ -14,6 +14,8 @@ import { handlePKitWarning } from '../handleWarning.js';
 
 const { details: X, quote: q } = assert;
 
+const isASCII = /^[a-zA-Z0-9_]+$/;
```

Comment:
ASCII has more than these symbols. 
```suggestion
const isAlphaNumeric = /^[a-zA-Z0-9_]+$/;
```

Reply:
Good point

Reply:
Chris and I discussed. No special constraints on these strings are needed, and in any case would need to be coordinated with similar constraints on the descriptions strings, which are equally unneeded.

Gone. Resolving.